content/blog/eng/php-day-2020.md

```diff
@@ -0,0 +1,69 @@
+Slides available: https://bit.ly/exceptions-ftw
+ 
+**Kubernetes for PHP developers by _Alessandro Lai_**
+
+A well-presented talk by my colleague Alessandro - Too much dashboard screenshot for me, but that's only because I am more dev than ops ;)
+
+**Technically Speaking: Improve your code with documentation by _Alexandra White_**
+
+Brilliantly presented by Alexandra! A tidbit that I brought home: there are moments to be (like) Shakespeare and others not. One of those when not to be Shakespeare is when writing documentation.
+I must say that this talk lighted my interest in technical writing.
+The cherry on the cake was that the presentation had subtitles, which should definitely become a de-facto standard at least for pre-recorded videos. Very inclusive!
+Slides: https://docs.google.com/presentation/d/1l7cjrouAGdsGae4nXyJLcGcool1l0zx5knXvL_4h04Q/edit#slide=id.g589a77319a_0_34
+
+**Looping the Loop with SPL Iterators by _Mark Baker_**
+
+A tour of the not-well-known iterators world and a little mea-culpa about the initially almost-not-existent SPL documentation! 🙂 
+Slides: https://www.slideshare.net/MarkBakerUK/looping-the-loop-with-spl-iterators/
+
+**Getting started with ReactPHP – Pushing real-time data to the browser by _Christian Lück_**
+
+This is the real React! Not that other little thingy made by Facebook! An introductive talk to the react world and the world is based on: async and event-driven programming and of course promises (https://music.youtube.com/watch?v=Ok1Z65OMi1Y&feature=gws_kp_track), 
+
+**Pragmatic Optimism: How PHP Conquered the Web by Pragmatic Optimism: How PHP Conquered the Web by _Samantha Quiñones_**
+
+I really enjoyed the final talk that was a historic retrospective of the web development, starting from Charles Babbage up to (fast forward) CGI, the Personal Home Page Tools - PHP Tools and its evolution.
+There was a point in this talk worth mentioning: sometimes people are ashamed to tell others they are using PHP because of (insert here any drawback you heard of - probably already fixed by now).
+The fact is: why we should be ashamed by declaring we are using PHP? Every problem could be sorted out by several approaches and tools, some better than others.
+Well, pragmatically, PHP is a tool that helps us solve our problems for a very long time and have demonstrated to listen to its community and to evolve.
+I liked the comparison with CoBOL: at the time it was published it was the perfect tool to program business applications, because the alternative were more or lest not-existent (unless you would go low-level).
+So, CoBol was the right tool at the right time. At still it is... (that's because big companies find cheaper to maintain their big underdocumented monoliths than re-engineering it, but this is another story).
```

Comment:
```suggestion
So, CoBol was the right tool at the right time. And still it is... (that's because big companies find cheaper to maintain their big underdocumented monoliths than re-engineering it, but this is another story).
```